### PR TITLE
I learnt loops and some basics of Java Script

### PR DESCRIPTION
**Pull request by ShubhamAgrahariOncehub**
Today's Work 10 January 2020.

### CODE REVIEW

**Comment by saxenarajat** [comment]
Why is there an empty `index.js` in the top directory?

**Comment by saxenarajat** [comment]
How are you running the files located in `Shubham-Agrahari/daily-learning-material` folder? Shouldn't they go in the `nodejs` folder?

**Comment by ShubhamAgrahariOncehub** [comment]
Sir its js file is also there in the folder,txt file is just for reference.
*Regards*
*Shubham Agrahari* | Software Trainee at OnceHub
Lets connect at | shubham.agrahari@oncehub.com | LinkedIn
<https://www.linkedin.com/in/shubham-agrahari-124006154>


________________________________________________________


   *Find out more about OnceHub:*
   - *Career Website <http://www.oncehub.com/careers/home#openPositions>  *
   *Website <http://www.oncehub.com/> | **Blog <http://blog.oncehub.com/>*
   * | **Facebook <http://www.facebook.com/OnceHub> **| Twitter
   <http://twitter.com/oncehub> | YouTube <https://www.youtube.com/OnceHub> *

*Watch the video: About OnceHub
   <https://www.youtube.com/watch?v=QuUtDqZ7Np8&feature=youtu.be>*

**Comment by ShubhamAgrahariOncehub** [comment]
Nageder Sir told us what we have done today to be sent only in
daily-learning-training folder,from today onwards I will send the file in
specific folder only.

*Regards*
*Shubham Agrahari* | Software Trainee at OnceHub
Lets connect at | shubham.agrahari@oncehub.com | LinkedIn
<https://www.linkedin.com/in/shubham-agrahari-124006154>


________________________________________________________


   *Find out more about OnceHub:*
   - *Career Website <http://www.oncehub.com/careers/home#openPositions>  *
   *Website <http://www.oncehub.com/> | **Blog <http://blog.oncehub.com/>*
   * | **Facebook <http://www.facebook.com/OnceHub> **| Twitter
   <http://twitter.com/oncehub> | YouTube <https://www.youtube.com/OnceHub> *

*Watch the video: About OnceHub
   <https://www.youtube.com/watch?v=QuUtDqZ7Np8&feature=youtu.be>*

**Comment by ShubhamAgrahariOncehub** [comment]
It's by mistake.
*Regards*
*Shubham Agrahari* | Software Trainee at OnceHub
Lets connect at | shubham.agrahari@oncehub.com | LinkedIn
<https://www.linkedin.com/in/shubham-agrahari-124006154>


________________________________________________________


   *Find out more about OnceHub:*
   - *Career Website <http://www.oncehub.com/careers/home#openPositions>  *
   *Website <http://www.oncehub.com/> | **Blog <http://blog.oncehub.com/>*
   * | **Facebook <http://www.facebook.com/OnceHub> **| Twitter
   <http://twitter.com/oncehub> | YouTube <https://www.youtube.com/OnceHub> *

*Watch the video: About OnceHub
   <https://www.youtube.com/watch?v=QuUtDqZ7Np8&feature=youtu.be>*

File: Shubham-Agrahari/daily-learning-material/operation.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+let isVerified= true
+let isloggedIn  = true
+let haspaymentToken =false
+let isGuest=true
```
Comment (so-ravindra) [inline Shubham-Agrahari/daily-learning-material/operation.js]: keep uniform spacing ( = )

**Pull request merged**